plugins/org.locationtech.udig.info/src/org/locationtech/udig/tool/info/internal/WMSDescribeLayer.java

```diff
@@ -202,7 +202,7 @@ public static LayerPointInfo info2( ILayer layer, ReferencedEnvelope query ) thr
 
         getmap.setBBox( bbox );
         String srs = CRS.toSRS(bbox.getCoordinateReferenceSystem() );
-        //getmap.setSRS( code != null ? code : srs );
+        getmap.setSRS( code != null ? code : srs );
```

Comment:
Can we cleanup commented line 211 with this pull request?

Reply:
I think yes. I let it there just because it was from the initial committer

Reply:
@fgdrf do you want to remove it?